src/net_processing.cpp

```diff
@@ -3037,6 +3046,63 @@ void PeerManagerImpl::ProcessHeadersMessage(CNode& pfrom, Peer& peer,
     return;
 }
 
+void PeerManagerImpl::ProcessInvalidTx(NodeId nodeid, const CTransactionRef& ptx, const TxValidationState& state,
```

Comment:
Nit: The var names differ from the declaration for `CTransactionsRef&` and `TxValidationState&`. Is there a reason for the `p` in `ptx`?

Reply:
`p` as in "pointer". I usually use `CMutableTransaction mtx` and `CTransaction tx`. Though I agree there's a mix...